lib/foreman_remote_execution_core/dispatcher.rb

```diff
@@ -0,0 +1,17 @@
+require 'foreman_tasks_core/runner/dispatcher'
+
+module ForemanRemoteExecutionCore
+  class Dispatcher < ::ForemanTasksCore::Runner::Dispatcher
+
+    def refresh_interval
+      # For the regular runner we don't want to let the user configure
+      #   the refresh interval so live output keeps working
+      @refresh_interval ||= if ForemanRemoteExecutionCore.settings[:async_ssh]
+                              ForemanRemoteExecutionCore.settings[:runner_refresh_interval].to_i
```

Comment:
Overriding the value without taking into account the config value, if specified, doesn't sound good. We could perhaps set the default for `runner_refresh_interval` in settings to `nil` and then apply the defaults here, if not specified. The difference would be we wouldn't overrride the value, if it was specified by the end user in settings